Src/GoogleApis.Auth/OAuth2/Requests/GoogleAuthorizationCodeRequestUrl.cs

```diff
@@ -49,6 +49,16 @@ public class GoogleAuthorizationCodeRequestUrl : AuthorizationCodeRequestUrl
         public string LoginHint { get; set; }
 
         /// <summary>
+        /// Gets or sets the Include Granted Scopes to determine if this authorization request should use
+        /// incremental authorization. If this is provided with the value <c>true</c>, and the authorization 

```

Comment:
I'm sorry you can just change it to:
/// Gets or sets the Include Granted Scopes to determine if this authorization request should use
/// incremental authorization (https://developers.google.com/+/web/api/rest/oauth#incremental-auth). If...

Or (the like you link you shared)
/// Gets or sets the Include Granted Scopes to determine if this authorization request should use
/// &lt;see href="https://developers.google.com/+/web/api/rest/oauth#incremental-auth"&gt;incremental authorization&lt;/&gt; (). If...